apps/web/src/app/(private)/_data-access/index.ts

```diff
@@ -84,3 +84,14 @@ export const getDocumentByPath = cache(
     return document
   },
 )
+
+export const getDocumentsAtCommit = cache(
```

Comment:
We also do this call in the sidebar. Can you use it there too?. Also let's start naming this cache method ending in `Cached` to easy greping